scripts/prepare/bundle.ts

```diff
@@ -79,7 +79,7 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
    */
   const nonPresetEntries = allEntries.filter((f) => !path.parse(f).name.includes('preset'));
 
-  const noExternal = [/^@vitest\/.+$/, ...extraNoExternal];
+  const noExternal = [...extraNoExternal];
```

Comment:
I added them ad dependencies:
https://github.com/storybookjs/storybook/blob/cb5b778aac3e050899864c38b2e2faa50aa4a47e/code/lib/test/package.json#L51-L52

Reply:
@kasperpeulen could you please comment here, what you think the correct course of action is here?

I added these dependencies, because AFAIK we must externalize these.
And the correct way to do that would be a peerDep or regular dep.
We can change it to what you think is best.